docker-build-v2/scripts/graph-deps.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+conan graph info \
```

Comment:
This prints out useful info (i.e. what all packages are being built and what packages are depending on other packages)

Reply:
If it's non-obvious enough to require a comment here, it probably warrants a proper comment in the file. Ditto for the other one

Reply:
Can do so, I mostly commented as I'm not sure how familiar people are with conan